Pod/Classes/WPMediaPickerViewController.m

```diff
@@ -309,6 +353,7 @@ - (void)refreshDataAnimated:(BOOL)animated
             strongSelf.collectionView.allowsSelection = YES;
             strongSelf.collectionView.allowsMultipleSelection = strongSelf.options.allowMultipleSelection;
             strongSelf.collectionView.scrollEnabled = YES;
+            strongSelf.collectionView.bounces = YES;
```

Comment:
`YES` is the default, so do we need this here?

Reply:
This can go away, I was try to find why our collection view doesn't bounce...